ipyvolume/pylab.py

```diff
@@ -675,67 +674,71 @@ def volshow(data, lighting=False, data_min=None, data_max=None,
     :param float data_min: minimum value to consider for data, if None, computed using np.nanmin
     :param float data_max: maximum value to consider for data, if None, computed using np.nanmax
     :parap int max_shape: maximum shape for the 3d cube, if larger, the data is reduced by skipping/slicing (data[::N]), set to None to disable.
-    :param tf: transfer function (or a default one)
+    :param tf_colornames: transfer function (or a default one)
```

Comment:
What about letting tf be a string or tuple, like [in this PR](https://github.com/jupyter-widgets/ipywidgets/pull/2182/files) where the icon can be a string or Icon widget.